src/referrals-queue/referrals-queue.component.tsx

```diff
@@ -31,17 +31,20 @@ export default function ReferralsQueue(props: ReferralsQueueProps) {
     .filter(r => matchQuery(r, ptQuery));
   const referralTypes = [...new Set(referrals.map(r => r.referral_type))];
   return (
-    <div className={`omrs-main-content ${styles.container}`}>
-      <div className="omrs-card omrs-margin-top-32 omrs-padding-16">
-        <div className={styles.greeting}>
+    <div className={`${styles.container}`}>
+      <div className="omrs-card omrs-margin-top-16 omrs-padding-16">
+        <div className="omrs-type-title-2">
           <Trans i18nKey="referrals-queue">Referrals Queue</Trans>
         </div>
-        <div>
-          <div className="omrs-input-group">
-            <label>
-              <Trans i18nKey="from">From</Trans>
+        <div className={styles.controlsContainer}>
+          <div className={`${styles.inputContainer}`}>
+            <div className={styles.dateInputContainer}>
```

Comment:
what's with the nested brackets and the quotes and the $ on line 40?

Reply:
Oops, cruft. That syntax is useful if you want to use dynamic and static class names together. The backticks are ES6 string interpolation. In brief, the syntaxes are:

```
className="some-static-name"
className={styles.anImportedClass}
className={`some-static-name ${styles.anImportedClass}`}
```